### PR TITLE
Update the readme file with the new name

### PR DESCRIPTION
**Pull request by victorhurdugaci**
Just a readme update

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,13 +4,13 @@ dotnet-watch
...
 
-Add `dotnet-watch` to the `tools` section of your `project.json` file:
+Add `Microsoft.DotNet.Watcher.Tools` to the `tools` section of your `project.json` file:
 
 ```
 {
 ...
   "tools": {
-    "dotnet-watch": "1.0.0-*"
+    "Microsoft.DotNet.Watcher.Tools": "1.0.0-*"
```
Comment (moozzyk) [inline README.md]: Probably need to add imports - at least for now.

Comment (muratg) [inline README.md]: Agreed, just test and I'm sure it won't work with imports for now.

**Comment by victorhurdugaci** [comment]
Updated

File: README.md [hunk 2]
```diff
@@ -4,13 +4,16 @@ dotnet-watch
...
+Add `Microsoft.DotNet.Watcher.Tools` to the `tools` section of your `project.json` file:
 
 ```
 {
 ...
   "tools": {
-    "dotnet-watch": "1.0.0-*"
+    "Microsoft.DotNet.Watcher.Tools": {
+      "version": "1.0.0-*",
+      "imports": "portable-net451+win8"
```
Comment (muratg) [inline README.md]: Is this enough?  cc @pakrym

Comment (victorhurdugaci) [inline README.md]: It works in tests and in the apps that I develop

Comment (muratg) [inline README.md]: Did you try across OS/FW/bitness spectrum?

Comment (victorhurdugaci) [inline README.md]: Yes

**Comment by muratg** [comment]
:shipit:

**Pull request merged**